[2024-06-04T10:40:41.870+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_python_dag.fetch_and_insert_into_elasticsearch_emplacement_stations manual__2024-06-04T10:40:41.040728+00:00 [queued]>
[2024-06-04T10:40:41.874+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_python_dag.fetch_and_insert_into_elasticsearch_emplacement_stations manual__2024-06-04T10:40:41.040728+00:00 [queued]>
[2024-06-04T10:40:41.875+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2024-06-04T10:40:42.013+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): fetch_and_insert_into_elasticsearch_emplacement_stations> on 2024-06-04 10:40:41.040728+00:00
[2024-06-04T10:40:42.016+0000] {standard_task_runner.py:57} INFO - Started process 668 to run task
[2024-06-04T10:40:42.019+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'my_python_dag', 'fetch_and_insert_into_elasticsearch_emplacement_stations', 'manual__2024-06-04T10:40:41.040728+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/fetch_from_data_sources.py', '--cfg-path', '/tmp/tmpobjn5x8q']
[2024-06-04T10:40:42.021+0000] {standard_task_runner.py:85} INFO - Job 100: Subtask fetch_and_insert_into_elasticsearch_emplacement_stations
[2024-06-04T10:40:42.044+0000] {task_command.py:415} INFO - Running <TaskInstance: my_python_dag.fetch_and_insert_into_elasticsearch_emplacement_stations manual__2024-06-04T10:40:41.040728+00:00 [running]> on host 9e4bf1cd7746
[2024-06-04T10:40:42.084+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_python_dag' AIRFLOW_CTX_TASK_ID='fetch_and_insert_into_elasticsearch_emplacement_stations' AIRFLOW_CTX_EXECUTION_DATE='2024-06-04T10:40:41.040728+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-04T10:40:41.040728+00:00'
[2024-06-04T10:40:42.086+0000] {logging_mixin.py:151} INFO - Fetching data from JSON
[2024-06-04T10:40:42.103+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/stations_functions.py", line 120, in fetch_and_insert_into_elasticsearch_emplacement_stations
    insert_into_elasticsearch(content)
  File "/opt/airflow/dags/stations_functions.py", line 104, in insert_into_elasticsearch
    client = Elasticsearch(
TypeError: __init__() got an unexpected keyword argument 'scheme'
[2024-06-04T10:40:42.109+0000] {taskinstance.py:1398} INFO - Marking task as UP_FOR_RETRY. dag_id=my_python_dag, task_id=fetch_and_insert_into_elasticsearch_emplacement_stations, execution_date=20240604T104041, start_date=20240604T104041, end_date=20240604T104042
[2024-06-04T10:40:42.114+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 100 for task fetch_and_insert_into_elasticsearch_emplacement_stations (__init__() got an unexpected keyword argument 'scheme'; 668)
[2024-06-04T10:40:42.158+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-06-04T10:40:42.169+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
